src/openbiolink/gui/gui.py

```diff
@@ -87,7 +87,8 @@ def _main(*args_lists):
                 for arg_list in args_lists:
                     openBioLink.main(arg_list)
 
-            args = [self.ARGS_LIST_GRAPH_CREATION, self.ARGS_LIST_TRAIN_TEST_SPLIT, self.ARGS_LIST_EVAL]
+            args = [self.ARGS_LIST_GLOBAL, self.ARGS_LIST_GRAPH_CREATION, self.ARGS_LIST_TRAIN_TEST_SPLIT, self.ARGS_LIST_EVAL]
+            print(args)
```

Comment:
I can't get the cl to set the correct working directory, the args list would be f.e.
`[['-p', 'C:/Users/Simon/Desktop/test'], ['generate', '--output-format', 'TSV', '--qual', 'hq', '--no-download', '--no-input', '--output-sep', 't'], [], []]`

and the error is:
`Usage: __main__.py [OPTIONS] COMMAND [ARGS]...
Try "__main__.py --help" for help.

Error: Missing command.`

I am working on my understanding of click, maybe this is easy to solve for you.

P.S. I just realized i forgot to remove the print argument, sorry.

Reply:
Solved in 940358c. I forgot to append the global arg list to each of the others